android/src/main/java/com/twilio/twilio_voice/BackgroundCallJavaActivity.java

```diff
@@ -220,4 +221,4 @@ protected void onDestroy() {
         deactivateSensor();
     }
 
-}
\ No newline at end of file
+}
```

Comment:
@shiburagi please remove these 2 lines, will merge changes into master for next twilio_voice update

Reply:
@shiburagi